src/components/FileMetadata/index.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+
+import { VersionFile } from '../../reducers/versions';
+import styles from './styles.module.scss';
+import { gettext } from '../../utils';
+
+type PublicProps = {
+  file: VersionFile;
+};
+
+const FileMetadataBase = ({ file }: PublicProps) => {
+  return (
+    <div className={styles.FileMetadata}>
+      <dl>
+        <dt>{gettext('Version')}</dt>
+        <dd className={styles.version}>{file.version}</dd>
+        <dt>{gettext('Size')}</dt>
+        <dd className={styles.size}>{file.size}</dd>
+        <dt>{gettext('SHA256 hash')}</dt>
+        <dd className={styles.sha256}>{file.sha256}</dd>
+        <dt>{gettext('Mimetype')}</dt>
```

Comment:
Is it `Mimetype` or `MIME type`?

Reply:
I used "Mimetype" as that's what's used in the current app:

![Screenshot 2019-03-11 14 29 56](https://user-images.githubusercontent.com/142755/54148197-2e614880-440a-11e9-97b0-0b0749ee3799.png)

I will change it to "MIME type", but maybe we can get @wagnerand's opinion on what he things the label for this data should be?